package/MDAnalysis/analysis/waterdynamics.py

```diff
@@ -0,0 +1,1057 @@
+other hand, a fast decay means a short permanence time::
+
+  import MDAnalysis
+  from MDAnalysis.analysis.waterdynamics import SP
+
+  u = MDAnalysis.Universe(pdb, trajectory)
+  selection = "byres name OH2 and sphzone 12.3 (resid 42 or resid 26 or resid 34 or resid 80) "
+  SP_analysis = SP(universe, selection, 0, 100, 20)
+  SP_analysis.run()
+  #now we print data ready to graph. The graph
+  #represents SP vs t
+  i=0
+  for sp in SP_analysis.timeseries:
+        print i +" "+ sp
+        i += 1
+
+.. Output
+
+Output
+------
+
+HBL
+~~~
+Hydrogen bond lifetime data is returned per window timestep, which is stored in
+:attr:`HBL.timeseries` (in all the following descriptions, # indicates comments that are not part of the output)::
+
+    results = [
+        [ # time t0
+            <HBL_c>, <HBL_i>
+        ],
+        [ # time t1
+            <HBL_c>, <HBL_i>
+        ],
+        ...
+     ]
+
+WOR
+~~~
+Water orientational relaxation data is returned per window timestep, which is stored in
+:attr:`WOR.timeseries`::
+
+    results = [
+        [ # time t0
+            <WOR_OH>, <WOR_HH>, <WOR_dip>
+        ],
+        [ # time t1
+            <WOR_OH>, <WOR_HH>, <WOR_dip>
+        ],
+        ...
+     ]
+
+AD
+~~~
+Angular distribution data is returned per vector, which is stored in
+:attr:`AD.graph`. In fact, AD returns a histogram::
+
+    results = [
+        [ # OH vector values
+          # the values are order in this way: <x_axis  y_axis>
+            <cos_theta0 ang_distr0>, <cos_theta1 ang_distr1>, ...
+        ],
+        [ # HH vector values
+            <cos_theta0 ang_distr0>, <cos_theta1 ang_distr1>, ...
+        ],
+        [ # dip vector values
+           <cos_theta0 ang_distr0>, <cos_theta1 ang_distr1>, ...
+        ],
+     ]
+
+MSD
+~~~
+Mean Square Displacement data is returned in a list, which each element represents a MSD value in its respective
+window timestep. Data is stored in :attr:`MSD.timeseries`::
+
+    results = [
+         #MSD values orders by window timestep 
+            <MSD_t0>, <MSD_t1>, ...        
+     ]
+
+SP
+~~
+Survival Probability data is returned in a list, which each element represents a MSD value in its respective
+window timestep. Data is stored in :attr:`SP.timeseries`::
+
+    results = [
+         # SP values order by window timestep
+            <SP_t0>, <SP_t1>, ...
+     ]
+
+
+
+Classes
+--------
+
+.. autoclass:: HBL
+   :members:
+   :inherited-members:
+
+.. autoclass:: WOR
+   :members:
+   :inherited-members:
+   
+.. autoclass:: AD
+   :members:
+   :inherited-members:
+
+.. autoclass:: MSD
+   :members:
+   :inherited-members:
+
+.. autoclass:: SP
+   :members:
+   :inherited-members:
+
+
+"""
+from MDAnalysis import *
+import MDAnalysis.analysis.hbonds
+import MDAnalysis.core.Selection
+import numpy
+import argparse
+import sys
+import multiprocessing
+import time
+import itertools
+
+class HBL:
```

Comment:
This is similar to analysis.hbonds.HydrogenBondAutoCorrel, so I think we will probably have to merge the two.  I'll read this through carefully and see if we do the exact same thing...